webui/src/components/ReplicasetList/ReplicasetList.js

```diff
@@ -116,7 +116,36 @@ const styles = {
   `
 };
 
-const prepareReplicasetList = dataSource => [...dataSource].sort((a, b) => b.uuid < a.uuid ? 1 : -1);
+const compareAliases = (aAlias, bAlias) => {
+  if (aAlias && !bAlias) {
+    return -1;
+  }
+
+  if (!aAlias && bAlias) {
+    return 1;
+  }
+
+  if (aAlias !== bAlias) {
+    return bAlias < aAlias ? 1 : -1;
+  }
+  return null;
+}
+
+const prepareReplicasetList = dataSource => [...dataSource].sort((a, b) => {
+  let result = compareAliases(a.alias, b.alias)
+  if (result) {
+    return result;
+  }
+
+  const a_leader = a.servers.find(server => server.uuid === a.master.uuid);
+  const b_leader = b.servers.find(server => server.uuid === b.master.uuid);
+
+  result = compareAliases(a_leader.alias, b_leader.alias)
+  if (result) {
+    return result;
+  }
+  return b.uuid < a.uuid ? 1 : -1;
```

Comment:
Logic of handling missing values is spread over 2 functions. What about smth like this?
```js
let a_value = a.alias || ''
let b_value = b.alias || ''
if (a_value === b_value) {
  a_value = a.servers... || ''
  b_value = ...
}
if (a_value === b_value) {
  a_value = a.uuid
  b_value = ...
}
return a_value < b_value ? -1 : 1
```